vk_layer_settings.txt

```diff
@@ -0,0 +1,136 @@
+# The settings in this file can be used to configure the behavior of
```

Comment:
Should this go inside some directory? sample or script?

Reply:
Wasn't sure where the best place was. I like sample